src/traces/scattergl/attributes.js

```diff
@@ -58,7 +57,7 @@ module.exports = overrideAll({
             description: 'Sets the style of the lines.'
         }
     },
-    marker: extendDeep({}, colorAttributes('marker'), {
+    marker: extendFlat({}, colorAttributes('marker'), {
```

Comment:
these attributes should be pretty well tested, but just to confirm this wasn't changing anything, I checked that `JSON.stringify(attrsDeep) === JSON.stringify(attrsFlat)` ✅